### PR TITLE
Fix Skipper DER Bug

### PR DESCRIPTION
**Pull request by CovenEsme**
Fixes bug were Skipper's "To the Sandship" dialog option opens up after beating Tentalus - even in DER.

Removes extra entrance to SSH after beating Tentalus (causes problems with DER).

### CODE REVIEW

File: patches.yaml [hunk 1]
```diff
@@ -4220,6 +4218,30 @@ F301_1: # Sand Sea Main Area
     room: 0
     objtype: STAG
     id: 0xFC6B
+  - name: Remove excess entrance to Sadship (layer 2)
```
Comment (lepelog) [inline patches.yaml:4221]:
```suggestion
  - name: Remove excess entrance to Sandship (layer 2)
```
typo

File: patches.yaml [hunk 2]
```diff
@@ -4220,6 +4218,30 @@ F301_1: # Sand Sea Main Area
...
+    type: objdelete
+    id: 0xFC98
+    layer: 2
+    room: 0
+    objtype: STAG
+  - name: Remove excess entrance to Sadship (layer 3)
```
Comment (lepelog) [inline patches.yaml:4227]:
```suggestion
  - name: Remove excess entrance to Sandship (layer 3)
```

File: patches.yaml [hunk 3]
```diff
@@ -4220,6 +4218,30 @@ F301_1: # Sand Sea Main Area
...
+    type: objdelete
+    id: 0xFCB7
+    layer: 3
+    room: 0
+    objtype: STAG
+  - name: Remove excess entrance to Sadship (layer 4)
```
Comment (lepelog) [inline patches.yaml:4233]:
```suggestion
  - name: Remove excess entrance to Sandship (layer 4)
```

File: patches.yaml [hunk 4]
```diff
@@ -4220,6 +4218,30 @@ F301_1: # Sand Sea Main Area
...
+    type: objdelete
+    id: 0xFCB7
+    layer: 4
+    room: 0
+    objtype: STAG
+  - name: Remove excess entrance to Sadship (layer 5)
```
Comment (lepelog) [inline patches.yaml:4239]:
```suggestion
  - name: Remove excess entrance to Sandship (layer 5)
```
typo

**Review by lepelog — CHANGES_REQUESTED** [review]
Looks good apart from the "typos", needs a test to make sure the sandhip dock loadzone works both before and after defating tentalus, when that boss is in another dungeon entrance

**Comment by Muzugalium** [comment]
Tested:
-Shooting sandship down- entered correct dungeon
-Completing shipyard and re-entering the docked sandship- entered correct dungeon
-Completing pirates and re-entering the docked sandship- enteered correct dungeon
-Completing sandship in AC entrance and re-entering docked sandship- entered correct dungeon

**Pull request merged**